sympy/matrices/matrices.py

```diff
@@ -994,12 +994,15 @@ def orthogonalize(cls, *vecs, **kwargs):
         """Apply the Gram-Schmidt orthogonalization procedure
         to vectors supplied in `vecs`.
 
-        Arguments
-        =========
+        Parameters
+        ==========
+
+        vecs :
+            vectors to be made orthogonal
 
-        vecs : vectors to be made orthogonal
```

Comment:
This could be left as-is (without a line break) and for `normalize` you could address the previous comments.

Reply:
@sidhantnagpal 
I think there is a clear difference between how docs render with and without line break,
but is first one preferable?
![image](https://user-images.githubusercontent.com/34944973/46253959-a2460f00-c4c2-11e8-95c9-fee19f1fcbe8.png)

![image](https://user-images.githubusercontent.com/34944973/46253973-e507e700-c4c2-11e8-8417-2ff9298d7bab.png)

Reply:
A newline between the two parameters `vecs` and `normalize` will make it clear I suppose.

Reply:
@sidhantnagpal 
Actually, I don't think that any additional newlines are recognized as-is.
I think the docs are parsed in some way, and only some formats are being accepted.

I think sympy uses numpy's formatting sheet, as you can see some copies of `docscrape.py`, `numpydoc.py` in `sympy/doc/ext`
And from the numpy's doc tutorial
https://numpydoc.readthedocs.io/en/latest/format.html#sections
I think either
```
param : type
    description
```
or
```
param :
    description
```
```
param : type
```
are correct forms.

Reply:
I think you might have to drop the colon as well if that is the case. See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html#example-numpy.

> ```
> name : type
>     description
> 
>     The description may span multiple lines. Following lines
>     should be indented to match the first line of the description.
>     The ": type" is optional.
> 
>     Multiple paragraphs are supported in parameter
>     descriptions.
> ```